console/v1/tests/consoleplugins.console.openshift.io/AAA_ungated.yaml

```diff
@@ -87,3 +87,176 @@ tests:
           i18n:
             loadType: Invalid
       expectedError: "Unsupported value: \"Invalid\""
+    - name: Should be able to create a ConsolePlugin with valid CSP configuration
+      initial: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - https://script1.com/
+          - directive: ImgSrc
+            values:
+            - https://img1.com/
+      expected: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - https://script1.com/
+          - directive: ImgSrc
+            values:
+            - https://img1.com/
+    - name: Should throw an error for duplicate CSP directive entries
+      initial: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - https://script1.com/
+          - directive: ScriptSrc
+            values:
+            - https://script1.com/
+      expectedError: "spec.contentSecurityPolicy[1]: Duplicate value: map[string]interface {}{\"directive\":\"ScriptSrc\""
```

Comment:
I asked you to include the rest of this error message for completeness, but it seems to have gone
It's odd seeing the unbalanced braces